content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst

```diff
@@ -36,3 +36,20 @@ process**, at the :guilabel:`Confirm Order` step.
 .. image:: payments/payments-checkout.png
    :align: center
    :alt: Payment provider selection at checkout
+
+eWallets and gift cards
+=======================
+
+Customers can pay with an **eWallet** or **gift cards** at the payment checkout step. To enable
+eWallets and gift cards, go to :menuselection:`Website --> Configuration --> Settings`, and in the
+**Shop - Products** section, enable :menuselection:`Discounts, Loyalty & Gift Card`.
```

Comment:
When checking out, customers can pay with an **eWallet** or **gift cards**. To enable these, go to :menuselection:`Website --> Configuration --> Settings`, and in **Shop-Products** section, enable :menuselection:`Discounts, Loyalty & Gift Card`.